debian/control

```diff
@@ -5,8 +5,6 @@ Maintainer: Eugene Blikh <bigbes@tarantool.org>
 Build-Depends: debhelper (>= 9), cdbs,
                cmake (>= 2.8),
                tarantool-dev (>= 1.10.0),
-               libsmall-dev,
-#              libmsgpuck-dev,
```

Comment:
Built package will depend on whether the libraries are installed within the build container due to [this logic](https://github.com/tarantool/memcached/blob/f5ce080a624134a4e7e05973e3723ef7a405b281/CMakeLists.txt#L37-L55). It should be solved somehow: say, using an option to choose a bundled/external library.

I would choose bundled libraries by default. It is convenient as for a programmer as well as for building our own packages. A downstream distro package build script may use an option to switch to an external library.

Reply:
Done.